apps/videos-admin/src/app/[locale]/(dashboard)/videos/[videoId]/_VideoView/Metadata/VideoInformation/VideoInfomation.tsx

```diff
@@ -0,0 +1,200 @@
+import { useMutation } from '@apollo/client'
+import Checkbox from '@mui/material/Checkbox'
+import FormControl from '@mui/material/FormControl'
+import FormControlLabel from '@mui/material/FormControlLabel'
+import InputLabel from '@mui/material/InputLabel'
+import MenuItem from '@mui/material/MenuItem'
+import Select from '@mui/material/Select'
+import Stack from '@mui/material/Stack'
+import TextField from '@mui/material/TextField'
+import { Form, Formik, FormikValues } from 'formik'
+import { graphql } from 'gql.tada'
+import { useParams } from 'next/navigation'
+import { useTranslations } from 'next-intl'
+import { ReactElement } from 'react'
+import { boolean, object, string } from 'yup'
+
+import { SaveButton } from '../../../../../../../../components/SaveButton'
+import { useAdminVideo } from '../../../../../../../../libs/useAdminVideo'
+
+const videoStatuses = [
+  { label: 'Published', value: 'published' },
+  { label: 'Unpublished', value: 'unpublished' }
+]
+
+const videoLabels = [
+  { label: 'Collection', value: 'collection' },
+  { label: 'Episode', value: 'episode' },
+  { label: 'Feature Film', value: 'featureFilm' },
+  { label: 'Segment', value: 'segment' },
+  { label: 'Series', value: 'series' },
+  { label: 'Short Film', value: 'shortFilm' },
+  { label: 'Trailer', value: 'trailer' },
+  { label: 'Behind The Scenes', value: 'behindTheScenes' }
+]
+
+interface VideoInformationProps {
+  isEdit: boolean
+}
+
+export const UPDATE_VIDEO_INFORMATION = graphql(`
+  mutation UpdateVideoInformation(
+    $titleInput: VideoTranslationUpdateInput!
+    $infoInput: VideoUpdateInput!
+  ) {
+    videoTitleUpdate(input: $titleInput) {
+      id
+      value
+    }
+    videoUpdate(input: $infoInput) {
+      id
+      label
+      slug
+      published
+      noIndex
+    }
+  }
+`)
+
+export function VideoInformation({
+  isEdit
+}: VideoInformationProps): ReactElement {
```

Comment:
```suggestion
export function VideoInformation(): ReactElement {
```

use the `useEdit` hook instead